catkit/hardware/iris_ao/iris_ao_controller.py

```diff
@@ -111,10 +128,12 @@ def _close(self):
         """Close connection safely."""
         try:
             self.log.info('Closing Iris AO.')
-            # Set IrisAO to zero
-            self.zero()
-            self.instrument.stdin.write(b'quit\n')
-            self.instrument.stdin.close()
+            # Since sending "quit" kills the proc a race condition exits between it quiting and a close() as it may
+            # exit before close() is called and thus cause close() to raise. This can even be true for an explicit call
+            # to stdin.flush() if the write buffer auto flushes.
+            # The above can leave the device hanging and unreachable. The safest option is to send the following signal
+            # which is gracefully handled on the C++ side.
+            self.instrument.send_signal(signal.CTRL_C_EVENT)
```

Comment:
@kjbrooks This will mess up jost, however, given the bucket of issues I've been having I'd imagine they would exits also on jost, which should be using this new C++ code.

 Jost isn't currently using this catkit interface, is it, it's still the following open PR, i.e., [this one](https://github.com/spacetelescope/jost-package/pull/15), right?

Reply:
Sorry, I missed this comment. Yes, Jost is not currently using catkit for the iris ao so we are good.